### DISCUSSION TITLE
Command line options for make-user to automate it

### DISCUSSION THREAD

**Discussion by arthurzenika** [postalserver/postal#2682] [Feature suggestions]
It would be nice to be able to automate the make-user postal command. 

Right now I found this hack : 

```bash
printf "test@email.com\nfirstname\nsurname\nsecretpassword" | docker-compose -p postal -f /opt/postal/install/docker-compose.yml exec -T web postal make-user
```

(the addition of `-T` is required and prevents using `| postal make-user` )

but it would be nicer to use the `postal make-user` with some command line options and maybe the password as a variable so that it doesn't stay in the `bash_history`

**Comment by deefdragon** [comment]
This would be very appreciated as its basically the one step preventing the creation of a helm chart for postal. (tho this hack may make it possible to do, its not ideal)

**Reply by arthurzenika** [reply]
hi @deefdragon is there anywhere we can follow the helm chart for postal ? We might have a use case for it...

**Reply by deefdragon** [reply]
Ive not started on it yet. I'm starting with verifying my Kubernetes deployment functions as I expect it to to begin with.

**Reply by arthurzenika** [reply]
Another problem with this script is that we cannot make "non-admin" users with this method. Any chance of seing that change ?
